src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -40,7 +40,6 @@ public class CertificateGenerator
         // Give the cert a grace period in case there's a time skew between machines
         private readonly TimeSpan _gracePeriod = TimeSpan.FromHours(1);
 
-        private const string _authorityCanonicalName = "DO_NOT_TRUST_WcfBridgeRootCA";
```

Comment:
The way it's done currently breaks encapsulation of the CertificateGenerator class. One should be initializing a certificate generator like so: 

```csharp
s_certificateGenerator = new CertificateGenerator() 
{ 
    CertificatePassword = config.BridgeCertificatePassword, 
    CrlUriBridgeHost = string.Format("http://{0}:{1}", config.BridgeHost, config.BridgePort), 
    CrlUriRelativePath = s_crlUriRelativePath, 
    ValidityPeriod = config.BridgeCertificateValidityPeriod 
}; 
```

My assertion is that we should be doing something similar here for `AuthorityCanonicalName` for initialization rather than have the `CertificateGenerator` "reach out" of the class to get it.